.github/cherry-pick.yml

```
@@ -0,0 +1,24 @@
+on:
+  pull_request:
+    branches:
+      - develop
+    types: ["closed"]
+
+jobs:
+  cherry_pick_release_v3_0:
+    runs-on: ubuntu-latest
+    name: Cherry pick into v3.0.0/develop
+    if: ${{ contains(github.event.pull_request.labels.*.name, 'v3.0.0/develop') && github.event.pull_request.merged == true }}
```

should this be 'develop' rather than 'v3.0.0/develop'? as the change would have been merged into `develop` branch

---

in fact, can the check just be:
```suggestion
    if: ${{ github.event.pull_request.merged == true }}
```
as this workflow should only be running against the `develop` branch, shouldn't it?

---

Yes, that is correct.